showcase/gapic-showcase/pom.xml

```diff
@@ -68,11 +67,11 @@
                 </goals>
                 <phase>generate-sources</phase>
                 <configuration>
-                  <executable>bazelisk</executable>
+                  <executable>bash</executable>
                   <arguments>
-                    <argument>run</argument>
-                    <argument>--experimental_proto_descriptor_sets_include_source_info</argument>
-                    <argument>//showcase:update_gapic</argument>
+                    <argument>../scripts/generate_component.sh</argument>
```

Comment:
I _think_ the purpose of this `update` profile is to replace all three modules.

Reply:
I added a small comment in the poms to indicate what the profile does